src/criterion.rs

```diff
@@ -51,6 +60,10 @@ impl<'a, 'b> Profiler for PProfProfiler<'a, 'b> {
             Output::Flamegraph(_) => "flamegraph.svg",
             #[cfg(feature = "protobuf")]
             Output::Protobuf => "profile.pb",
+            // This is `""` but not `unreachable!()`, because `unreachable!()`
+            // will result in another compile error, so that the user may not
```

Comment:
Can really `unreachable!()` cause compile error? I do not realize it.

Reply:
If the only branch is "unreachable!()", the type of "filename" will be resolved as `()`, and it's illegal to pass `filename` to `benchmark_dir.join`.

Though,  adding a redundant type annotation `: &str` to the `filename` will fix this issue, it will also bring a big warning (because the following statement is unreachable statement in that case), so I prefer to return an empty string, and let the `compile_error` prints the only error.